.classpath

```diff
@@ -24,7 +24,6 @@
 			<attribute name="gradle_used_by_scope" value="test"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="src" path="src/test/skript"/>
```

Comment:
Did you mean to do this?

Reply:
It's eclipse, and it was rewritten to be updated because no one uses eclipse.

Reply:
I do use Eclipse, and this probably makes test scripts source folder not visible.